### PR TITLE
Create generate_report.gs

### PR DESCRIPTION
**Pull request by siddhimahawar**

### CODE REVIEW

File: generate_report.gs [hunk 1]
```diff
@@ -0,0 +1,87 @@
+/**
+* Generates the report of most executed script in google sheets
+* @param {Object} mostExecutedScript is the array of projectIds  
+* @param {String} title is the heading for the report sheet
+* @param {Number} limitResponse is the maximum number of responses in the sheet
+* @return {String} URL of generated report
+*/
+function generateReportForMostExecutedScript(mostExecutedScript, title, limitResponse) {  
+  var timeStamp, dateTimeAsString, rowContent;
```
Comment (shikhagupta1212) [inline generate_report.gs]: one var declaration per line and you could have initialized in the same line

File: generate_report.gs [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+/**
+* Generates the report of most active users in google sheets  
+* @param {Object} mostActiveUser is the array of userIds
+* @param {String} title is the heading for the report sheet
+* @param {Number} limitResponse is the maximum number of responses in the sheet
+* @return {String} URL of generated report
+*/
+function generateReportForMostActiveUsers(mostActiveUser, title, limitResponse) {  
+  var timeStamp, dateTimeAsString, rowContent;
```
Comment (shikhagupta1212) [inline generate_report.gs]: same as above and can you reuse the code instead of copy pasting same code everywhere

Comment (siddhimahawar) [reply generate_report.gs]: Reports have different format.
That's why i kept different functions for each report type

File: generate_report.gs (hunk 1 above)
Comment (siddhimahawar) [reply generate_report.gs]: Updated

File: generate_report.gs [hunk 3]
```diff
@@ -35,12 +35,12 @@ function generateReportForMostExecutedScript(mostExecutedScript, title, limitRes
 * @param {Number} limitResponse is the maximum number of responses in the sheet
 * @return {String} URL of generated report
 */
-function generateReportForMostActiveUsers(mostActiveUser, title, limitResponse) {  
+function generateReportForMostActiveUsers(mostActiveUser, timestamp_header, limitResponse) {
```
Comment (shikhagupta1212) [inline generate_report.gs:39]: variable names in camelcase

**Review by shikhagupta1212 — APPROVED** [review]

File: generate_report.gs (hunk 3 above)
Comment (siddhimahawar) [reply generate_report.gs:39]: Updated

**Pull request merged**